### PR TITLE
Monitor live test fix

### PR DESCRIPTION
**Pull request by rakshith91**
1. Fixes https://github.com/Azure/azure-sdk-for-python/issues/22100
2. Remove py2 support
3. Fix live test build

### CODE REVIEW

File: sdk/monitor/azure-monitor-query/dev_requirements.txt [hunk 1]
```diff
@@ -3,4 +3,4 @@
 -e ../../core/azure-core
 -e ../../identity/azure-identity
 azure-mgmt-loganalytics
-aiohttp>=3.0; python_version >= '3.5'
+aiohttp>=3.0
```
[the remark this answers is not in the document]
Comment (rakshith91) [reply sdk/monitor/azure-monitor-query/dev_requirements.txt:6]: so what the line means is "install aiohttp only if py version is > 3.5" - now we can assume that py version is 3.6 and above :)

**Review by srnagar — APPROVED** [review]
lgtm

**Pull request merged**